Sources/Extensions/UIScreen+safeScale.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIScreen+safeScale.swift
+//  Rover
+//
+//  Created by Andrew Clunis on 2020-10-29.
+//  Copyright © 2020 Rover Labs Inc. All rights reserved.
+//
+
+import UIKit
+
+extension UIScreen {
+    var safeScale: CGFloat {
+        let value = self.scale
+        // in some cases, we have seen it occur that scale has returned 0. In that event, return a sane value.
+        return value == 0 ? 3.0 : value
```

Comment:
Have you tested this to see the effect? What happens on a device with a 2x screen scale?

Reply:
I have done some further testing, have a look at PR description.

Reply:
Awesome, thanks!